connectivity/connectivity-demos/processes/rest/uiPath.mod

```diff
@@ -181,10 +183,8 @@ for(ReleaseDto rel : in.releases){
   }
 }
 for(RobotDto robo : in.robots){
-  if (robo.type == TypeEnum.UNATTENDED){
-  	ivy.log.info("robot in use "+robo);
```

Comment:
here's the improvement: no longer do we need to filter objects in pure java, since we already got the pre-filtered robots that are capable of executing our request